moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -257,9 +256,11 @@ const ProjectComponentsMapEdit = ({
    * @param {String} newValue - The new value from the autocomplete selector
    */
   const handleComponentSubtypeSelect = (e, newValue) => {
-    setSelectedComponentSubtype(
-      (newValue ?? e.target.value ?? "").toLowerCase()
```

Comment:
I removed `e.target.value` because our `<li>` does not have a value attribute and e.target.value is `0`. If there wasn't a newValue, we would be doing `0.toLowerCase()` which results in an error.